pkg/common/common.go

```diff
@@ -47,6 +48,13 @@ const (
 	ImporterAccessKeyID = "IMPORTER_ACCESS_KEY_ID"
 	// ImporterSecretKey provides a constant to capture our env variable "IMPORTER_SECRET_KEY"
 	ImporterSecretKey = "IMPORTER_SECRET_KEY"
+	// ImporterContent provides a constant to capture our env variable "IMPORTER_CONTENT"
+	ImporterContent = "IMPORTER_CONTENT"
+
+	// ImporterContentKubvirt (the default content) - copy the endpoint into an image.
+	ImporterContentKubvirt = "kubevirt"
```

Comment:
Type on L#55,  s/Kubvirt/Kubevirt/

Reply:
Irrelevant anymore after rebasing on top of #520.